new.go

```diff
@@ -248,6 +248,13 @@ func newBuilder(ctx context.Context, store storage.Store, options BuilderOptions
 		defer src.Close()
 	}
 
+	// If the pull command was used, we only pull the image,
+	// we don't create a container.
+	if options.ImageOnly {
+		fmt.Printf("%s\n", imageID)
```

Comment:
I think this should be in the pull.go

Reply:
The reason I did this hear is this function return a builder object.  That object isn't created until several lines beyond this point.  However, I probably could create a builder object here containing only the image id and return that .  I'll give that a go.